nixos/modules/services/misc/autobrr.nix

```
@@ -0,0 +1,70 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.autobrr;
+in
+{
+  options = {
+    services.autobrr = {
+      enable = mkEnableOption (lib.mdDoc "Autobrr");
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc "Open ports in the firewall for the Autobrr web interface.";
+      };
+
+      configureNginx = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
+        description = lib.mdDoc "Configure nginx as a reverse proxy for Autobrr-web and the Go service.";
+      };
+
+      package = mkPackageOption pkgs "autobrr" { };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.autobrr = {
+      description = "Autobrr";
+      after = [ "syslog.target" "network-online.target" ];
+      wantedBy = [ "multi-user.target" ];
+
+      serviceConfig = {
+        Type = "simple";
+        DynamicUser = true;
+        StateDirectory = "autobrr";
+        StateDirectoryMode = "0700";
+        ExecStart = "${pkgs.autobrr}/bin/autobrr --config '${WorkingDirectory}'";
```

Where is `WorkingDirectory` defined?

---

And please use `lib.getExe`:

```suggestion
        ExecStart = "${lib.getExe pkgs.autobrr} --config '${WorkingDirectory}'";
```